.cache/a1/8ed84832cc4f7e2e279294a4dc8a84.json

```diff
@@ -0,0 +1 @@
+{"id":"css/style.scss","dependencies":[{"name":"/Users/maja/Desktop/Projects/Travel_journal/css/style.scss","includedInParent":true,"mtime":1598980640694},{"name":"_css_loader","parent":"/Users/maja/Desktop/Projects/Travel_journal/css/style.scss","resolved":"/usr/local/lib/node_modules/parcel-bundler/src/builtins/css-loader.js"},{"name":"/Users/maja/Desktop/Projects/Travel_journal/package.json","includedInParent":true,"mtime":1598980640933}],"generated":{"css":"h1 {\n  color: red;\n}","js":"var reloadCSS = require('_css_loader');\n\nmodule.hot.dispose(reloadCSS);\nmodule.hot.accept(reloadCSS);"},"sourceMaps":{"css":{"mappings":[{"source":"css/style.scss","name":null,"original":{"line":1,"column":0},"generated":{"line":1,"column":0}},{"source":"css/style.scss","name":null,"original":{"line":2,"column":4},"generated":{"line":2,"column":2}}],"sources":{"css/style.scss":"h1 {\n    color: red;\n}"},"lineCount":null}},"error":null,"hash":"5866bee3d2d766fa50e75645b0e6a7e3","cacheData":{"env":{}}}
```

Comment:
You forgot to add `.gitignore` to this project. Where you need to add 

```
.cache
dist
node_modules
```

This is an essential thing to any project that has `package.json` because those files are not meant to be shared. Each person will create them locally.

After you create the `.gitignore` file, you need to remove those folders from git (while keeping them in your computer):

```

git rm -r --cached node_modules
git rm -r --cached dist
git rm -r --cached .cache

git commit -m 'Remove node_modules, dist and .cache'

git push
```

Come back here and you should no longer see them in git

Reply:
I added a global .gitignore and put ds_store in it, the way I understood it is creates a .gitignore in all the local repositories?

Reply:
Ahh that's why you asked me about it the other day.
Usually we shouldn't relly on global gitignore because each project has different neededs and it's a file to be shared with the project members. So it must be local to each repo to ensure everyone has the same configs. does it make sense?

Reply:
Totaly, the way I initially understood it was that the global file would be added to each repo (which sounded nice 😄)

Reply:
Oh you forgot to commit `.gitignore` too